src/authorizations/components/CustomApiTokenOverlay.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {FC} from 'react'
+
+// Components
+import {Overlay} from '@influxdata/clockface'
+
+interface OwnProps {
+  onClose: () => void
+}
+
+const CustomApiTokenOverlay: FC<OwnProps> = props => {
+  const handleDismiss = () => {
+    props.onClose()
+  }
+  return (
+    <Overlay.Container maxWidth={500}>
+      <Overlay.Header
+        title="Generate a Personal Api Token"
+        onDismiss={handleDismiss}
+      />
+    </Overlay.Container>
+  )
+}
+
+export default CustomApiTokenOverlay
```

Comment:
can you make this a named export please?

named exports automatically get tree-shaken by webpack which reduces bundle size. It also removes ambiguities when importing